doc/changes/changes49.xml

```diff
@@ -0,0 +1,1196 @@
+<Heading>New and updated packages since &GAP; 4.8.10</Heading>
+
+There were 132 packages redistributed together with &GAP; 4.8.10.
+The &GAP; 4.9.1 distribution includes 134 packages, including numerous
+updates of previously redistributed packages, and some major changes
+outlined below.
+<P/>
+
+The libraries of small, primitive and transitive groups which previously
+were an integral part of &GAP; were split into three separate packages
+<URL Text="PrimgGrp">http://gap-packages.github.io/primgrp/</URL>,
+<URL Text="SmallGrp">https://gap-packages.github.io/smallgrp/</URL> and
+<URL Text="TransGrp">http://www.math.colostate.edu/~hulpke/transgrp/</URL>:
+<List>
+<Item>
+<Package>PrimGrp</Package> package by Alexander Hulpke, Colva M. Roney-Dougal
+and Christopher Russell provides the library of primitive permutation
+groups which includes, up to permutation isomorphism (i.e., up to conjugacy
+in the corresponding symmetric group), all primitive permutation groups of
+degree &lt; 4096.
+</Item>
+<Item>
+<Package>SmallGrp</Package> package by Bettina Eick, Hans Ulrich Besche and
+Eamonn O’Brien provides the library of groups of certain <Q>small</Q>
+orders. The groups are sorted by their orders and they are listed up to
+isomorphism; that is, for each of the available orders a complete and
+irredundant list of isomorphism type representatives of groups is given.
+</Item>
+<Item>
+<Package>TransGrp</Package> package by Alexander Hulpke provides
+the library of transitive groups, with an optional download of the
+library of transitive groups of degree 32.
+</Item>
+</List>
+For backwards compatibility, these are required packages in &GAP; 4.9
+(i.e., &GAP; will not start without them). We plan to change this for
+&GAP; 4.10, once all packages which currently implicitly rely on these
+new packages had time to add explicit dependencies on them
+(<URL Text="#1650">https://github.com/gap-system/gap/pull/1650</URL>,
+<URL Text="#1714">https://github.com/gap-system/gap/pull/1714</URL>).
+<P/>
+
+New package <Package>ZeroMQInterface</Package> by Markus Pfeiffer, which
+provides both low-level bindings as well as some higher level interfaces
+for the <URL Text="ZeroMQ">http://zeromq.org/</URL> message passing
+library for &GAP; and &HPCGAP; enabling lightweight distributed
+computation, has been added for the redistribution.
+<P/>
+
+<Package>HAPprime</Package> package by Paul Smith is no longer redistributed
+with &GAP;. Part of the code has been incorporated into the <Package>HAP</Package>
+package. <Package>HAPprime</Package> source code repository could be found at
+<URL>https://github.com/gap-packages/happrime</URL>.
+<P/>
+
+Also, <Package>ParGAP</Package> package by Gene Cooperman 
+is no longer redistributed with &GAP; (see
+<URL Text="this announcement">https://mail.gap-system.org/pipermail/gap/2018-March/001082.html </URL>). 
+<Package>ParGAP</Package> source code repository could be found at
```

Comment:
could -> can; all the same remarks as for HAPPrime apply, though here one could write: "containing the code of the last distributed version, plus some first fixes needed for compatibility for GAP 4.9"?

Reply:
We could also insert: "If somebody is interested in repairing this package and taking over its maintenance, so that it can be distributed again, please contact the GAP team." ?